lib/factory_bot/factory.rb

```diff
@@ -140,6 +142,8 @@ def compiled_constructor
     private
 
     def assert_valid_options(options)
+      raise ArgumentError, "Expected a Hash of options for factory '#{name.inspect}', got '#{options.inspect}'" unless options.is_a?(Hash)
```

Comment:
Unless things have changed, thoughtbot prefer 80 char lines and to avoid conditional surprises

```suggestion
      if !options.is_a?(Hash)
        raise ArgumentError,
          "Expected a Hash of options for factory '#{name.inspect}', got '#{options.inspect}'"
      end
```